docs/manual/java/gettingstarted/WhatsNext.md

```diff
@@ -0,0 +1,14 @@
+# What's Next?
+
+After working through "Getting started with Lagom", follow the path that make the most sense to you:
+
+* Learn how to integrate with Eclipse:
+    * [[Import an existing Maven project into Eclipse|EclipseMavenInt]] 
+    * [[Create a new Eclipse project using the Lagom Maven archetype|EclipseMavenNew]]
+    * [[Import an existing sbt build into Eclipse|EclipseSbt]]
```

Comment:
Sorry, I see now that I left the crucial word out of my original question. Should have said "Will there be a page on creating a new _sbt_ project in Eclipse?"